src/TelegramLoggerHandler.php

```diff
@@ -57,7 +58,7 @@ public function __construct(string $logLevel)
      * @param array $record
      * @return void
      */
-    protected function write(array $record): void
+    protected function write(LogRecord $record): void
```

Comment:
It breaks compatibility with monolog 2.* (it has such interface, but it hasn't implementation), so it's needed to remove it from compatible packages in composer.json. As well, as Laravel 7 and 8, that requires monolog 2.*.

(Or it could be changed to array|LogRecord, but it'll break compatibility with Laravel before 9, cause union types is php8 feature. So, result the same: no monolog 2 and laravel before 9 support => array isn't neccessary for this typehint.)